tests/builder/axes_test.py

```diff
@@ -740,3 +740,47 @@ def test_virtual_masters_extend_min_max_for_unmapped_axis(ufo_module, datadir):
     assert [
         cp.value for cp in font2.customParameters if cp.name == "Virtual Master"
     ] == virtual_masters
+
+
+def test_axis_mappings_different_interpretations(ufo_module, datadir):
+    # https://github.com/googlefonts/glyphsLib/issues/859
+    font = GSFont(datadir.join("gf", "Rubik.glyphs"))  # Saved under Glyphs 3079
```

Comment:
this Rubik.glyphs seems to already have a correct user->design mapping even if it was saved with an old version:

https://github.com/googlefonts/glyphsLib/blob/1cb4fc5ae2cf385df95d2b7768e7ab4eb60a5ac3/tests/data/gf/Rubik.glyphs#L49-L59

Reply:
Right, because it was a format 2 file.